src/entities/Article.ts

```diff
@@ -8,7 +8,7 @@ export class Article {
   id: string;
 
   @Field()
-  @Property({ unique: true })
+  @Property()
```

Comment:
Why can't this be `unique`

Reply:
I believe it caused errors when trying to embed publication object within the article but not 100% sure. I remember we had a conversation about this and deemed it no longer necessary for it to be true for some reason...